library/src/com/google/maps/android/ui/RotationLayout.java

```diff
@@ -26,7 +26,7 @@
  * <p/>
  * May not work with padding.
  */
-class RotationLayout extends FrameLayout {
+public class RotationLayout extends FrameLayout {
```

Comment:
it was a lint error - since this class is used in `layout.xml` file, lint requires it to be public